ovn-org/ovn-kubernetes
# Separate multicast, refactor policy_test

Move multicast functions to a separate file (no changes)

policy_test refactoring:
- add functions to generate defaultDenyACL and gressACLs,
and their stale versions to test sync
- add functions to generate network policy and testPod
- move common test constants to ginkgo.Describe level
- move expected data generation right before checking this data instead
of beginning of the test
- define startOvn function to unify fakeOvn start sequence
- remove kNetworkPolicy since it doesn't have any field and all its
  methods can be simple functions
- update "ACL logging for network policies" Context to use common
  functions instead of defining its own
- inline some "ACL logging for network policies" Context function for
  better readability

Update "reconciles an existing networkPolicy with a pod and namespace
selector in another namespace from empty db" test to check pod data
with properly populating ls cache and checking existing data has
getExpectedDataPodsAndSwitches.
